contract_discount/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+Contract Discount
+=================
+
+This module was written to extend the functionality of contracts to support set a discount on contract invoice lines.
+
+Usage
+=====
+
```

Comment:
Go to settings -> Configuration -> Sales and activate "Allow setting a discount on the sales order lines" to see discount in contracts.